core/ui/src/components/Form/Select/index.tsx

```diff
@@ -0,0 +1,104 @@
+import React from 'react'
```

Comment:
A organização das pastas deve ser diferente, lembra? padronizada, então seria `core/ui/src/Form/components/Select/index.tsx`

Reply:
na verdade foi dada essa estrutura:
- components
--- Window
---- components
----- Header
------ index.tsx
------ styles.ts
----- SideBar
------- index.tsx
------- styles.ts
--- SideBar
---- index.tsx
---- styles.ts

e que todos deveriam ficar dentro de componentes.

Reply:
Exatamente, vc acabou de repetir o que eu falei, observe como nessa estrutura de arquivos não existe `components/Window/Header`, mas sim `components/Window/components/Header`, sendo que esse Header deve ser usado exclusivamente pelo Window, se esse Header for usado em algum lugar mais ele deve estar em `components/Header`. Essa é uma estrutura que propõe escalabilidade e não é de minha cabeça, é o que sempre usamos baseado nesse artigo: https://alexmngn.medium.com/how-to-better-organize-your-react-applications-2fd3ea1920f1

Reply:
mas porque form fica fora da pasta components?

Reply:
resolvido